src/Microsoft.Dnx.Runtime.Sources/Impl/Constants.cs

```diff
@@ -20,11 +20,10 @@ internal static class Constants
         public const string WebConfigRuntimePath = "runtime-path";
         public const string BootstrapperHostName = RuntimeShortName + ".host";
         public const string BootstrapperClrName = RuntimeShortName + ".clr";
-        public const string ProjectJsonAppDomainDataKey = ".dnx.clr.DomainManager.projectJson";
-        public const string ProjectJsonPathAppDomainDataKey = ".dnx.clr.DomainManager.projectJsonPath";
 
         public const int LockFileVersion = 1;
 
         public static readonly TimeSpan RegexMatchTimeout = TimeSpan.FromSeconds(10);
+        public static readonly string AppConfigurationFileName = "App.Config";
```

Comment:
:+1: though I don't actually know what it should be. Regardless, this is only used on case-insensitive file systems (because it's Windows-only). I'll change it to lower-case for consistency though.